prep-all.sh

```
@@ -49,14 +49,14 @@
 nix develop -i ${keep[@]} -c rainix-rs-prelude
 nix develop -i ${keep[@]} -c raindex-prelude
 
-echo "Setting up UI components..."
-nix develop -i ${keep[@]} .#tauri-shell -c ob-tauri-prelude
-nix develop -i ${keep[@]} .#tauri-shell -c ob-ui-components-prelude
+# echo "Setting up UI components..."
+# nix develop -i ${keep[@]} .#tauri-shell -c ob-tauri-prelude
+# nix develop -i ${keep[@]} .#tauri-shell -c ob-ui-components-prelude
 
 echo "Building packages..."
 nix develop -i ${keep[@]} -c bash -c '(npm run build -w @rainlanguage/orderbook)'
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)